submissions/nallikaea/html-css-popup/index.html

```diff
@@ -17,120 +17,120 @@
         <a  class="text-nav-item" href="#">Mail</a>
       </li>
       <li id="nav-popup">
```

Comment:
The list item can"t include all markup. One item can include only text (or text and picture). Could you please fix it? 
Each item in popup should be the list item. 
Id "nav-popup" should be the menu (with structure  nav > ul > li )